DEPLOYMENT.md

```diff
@@ -28,7 +27,7 @@ mv addressfinder-woo addressfinder-woocommerce-svn
 cd addressfinder-woocommerce-svn
 
 cp ../addressfinder-woocommerce/dist/addressfinder.min.js trunk/addressfinder.js
-cp ../addressfinder-woocommerce/dist/addressfinder.js.map trunk/addressfinder.js.map
+cp ../addressfinder-woocommerce/dist/addressfinder.min.js.map trunk/addressfinder.min.js.map
```

Comment:
So was there anything at `../addressfinder-woocommerce/dist/addressfinder.js.map` that we were copying to `trunk/addressfinder.js.map`? What is this source map used for?

Reply:
[What are source maps](https://www.rapid7.com/blog/post/2017/05/24/what-are-javascript-source-maps/)?

`addressfinder.min.js` has hardcoded source map that it links to - `addressfinder.min.js.map`. We rename the `addressfinder.min.js` when we upload it to the wp store but we should not rename the source map because it's name is hardcoded in the js file.